api/api.cc

```diff
@@ -40,22 +40,22 @@ static std::unique_ptr<reply> exception_reply(std::exception_ptr eptr) {
     try {
         std::rethrow_exception(eptr);
     } catch (const replica::no_such_keyspace& ex) {
-        throw bad_param_exception(ex.what());
+        throw httpd::bad_param_exception(ex.what());
```

Comment:
I would say that all api/.cc files (_*.cc_ files) deserve having `using namespace seastar::httpd` instead of explicit namespace specification everywhere